### PR TITLE
fix: publish to npm correctly

### PR DESCRIPTION
**Pull request by djamaile**

### CODE REVIEW

File: .github/workflows/publish.yaml [hunk 1]
```diff
@@ -73,7 +73,7 @@ jobs:
 
       - name: publish
         run: |
-          pnpm config set -H 'npmAuthToken' "${{secrets.NPM_TOKEN}}"
-          pnpm npm publish --access public --tolerate-republish
+          pnpm config set npmAuthToken "${{secrets.NPM_TOKEN}}"
+          pnpm publish --force
```
Comment (mitchhentgesspotify) [inline .github/workflows/publish.yaml:77]: Why is `--force` needed here?

**Review by mitchhentgesspotify — CHANGES_REQUESTED** [review]

File: .github/workflows/publish.yaml (hunk 1 above)
Comment (djamaile) [reply .github/workflows/publish.yaml:77]: So, we can republish 😄 , basically equal to `--tolerate-republish`

Comment (mitchhentgesspotify) [reply .github/workflows/publish.yaml:77]: Ok, but when would we need to republish? Is a different part of CI pushing a version number, and this comes up later and pushes the code? 🤔

Comment (djamaile) [reply .github/workflows/publish.yaml:77]: Nope, I think its there as a safeguard to remove certain stuff from a release if needed and you can't publish a new version for it. I copied it over from backstage itself but we can remove it if you want

Comment (mitchhentgesspotify) [reply .github/workflows/publish.yaml:77]: [I found the PR where Backstage added `--tolerate-republish`](https://github.com/backstage/backstage/pull/13848) - maybe it's for re-publishing a _tag_ (so they can keep updating what `@nightly` refers to?)

Either way, you're right that `--force` here is like `--tolerate-republish`, and I missed that you meant that as in "hey, we had it before, so it's more of a behaviour change to _remove_ it".

So, 👍 either way, whether you want to keep it or remove it.

**Review by mitchhentgesspotify — APPROVED** [review]

**Pull request merged**